### PR TITLE
Harmony, disk space and small tidies

### PR DESCRIPTION
**Pull request by bokkypoobah**

### CODE REVIEW

File: clients.asciidoc [hunk 1]
```diff
@@ -68,14 +67,14 @@ Whether or not you choose to run a full node, you will probably want to run a pu
...
-* Deploying contracts or making transactions requires test ether, which has no value and can be acquired for free from several sources.
+* Deploying contracts or making transactions requires test ether, which has no value and can be acquired for free from several "faucets".
 * Testnets are public blockchains with many other users and contracts, running "live."
 
 *Disadvantages:*
 
 * You can't use "real" money on a testnet, it runs on test ether.
 * Consequently, you can't test security against real adversaries, as there is nothing at stake.
-* There are some aspects of a public blockchain that you cannot test realistically on testnet. For example, transaction fees, although necessary to send transactions, are not a consideration on testnet since gas is free.
+* There are some aspects of a public blockchain that you cannot test realistically on testnet. For example, transaction fees, although necessary to send transactions, are not a consideration on testnet since gas is free. And the testnets do not experience network congestions like the public network.
```
Comment (fjrojasgarcia) [inline clients.asciidoc:77]: Here I would add something like _"libraries that have not been previously deployed as well as the absence of a World State (in the Blockchain) required for certain tests"_.

Comment (fjrojasgarcia) [reply clients.asciidoc:77]: I would say, _"on testnet ether is free"_ rather than _"on testnet gas is free"_.

Comment (bokkypoobah) [reply clients.asciidoc:77]: Merge my changes and I'll add your suggestions

Comment (fjrojasgarcia) [reply clients.asciidoc:77]: If you add changes to the branch (bokkypoobah:develop) of this PR, automatically appears in it.

**Pull request merged**